result/egressonlytracking.yaml

```diff
@@ -0,0 +1,471 @@
+components:
+  schemas:
+    EgressOnlyTracking.Metrics.Request:
+      description: >-
+        The container for a egress only tracking metric request.
+      type: object
+      properties:
+        port_names:
+          description: |-
+            Egress only tracking metrics will be retrieved for these port names.
+          type: array
+          items:
+            type: string
+          x-constraint:
+          - "/components/schemas/EgressOnlyTacking/properties/port_name"
+          x-field-uid: 1
+        metric_names:
```

Comment:
Why do we have this field. Note there is no per flow stats anymore.

Reply:
Done.